examples/regression/3-scientific-articles/plot_kim2020_simulations.py

```diff
@@ -68,6 +69,7 @@ def get_X_y() -> Tuple[NDArray, NDArray]:
     zip_folder = "BlogFeedback.zip"
     csv_file = "blogData_train.csv"
     url = website + page + folder + zip_folder
+    ssl._create_default_https_context = ssl._create_unverified_context
```

Comment:
I did this, but not sure that it's a best practice. Maybe keep it as an issue for later?

Reply:
Could you put the link of the new issue?

Reply:
https://github.com/scikit-learn-contrib/MAPIE/issues/378